src/test/java/com/adyen/PaymentTest.java

```diff
@@ -468,7 +477,7 @@ public void TestByteArrayToJSONString() throws Exception {
         
         payment.authorise(paymentRequest);
         // Unicode representation of the string
-        String expected = "\"mpiData\":{\"cavv\":\"AQIDBAUGBwgJCgsMDQ4PEBESExQ\\u003d\"}";
+        String expected = "\"mpiData\":{\"cavv\":\"QVFJREJBVUdCd2dKQ2dzTURRNFBFQkVTRXhRPQ==\"}";
```

Comment:
This is expectation is not correct. It's based on double base64 encode.

Reply:
Right, good catch! Completely forgot about that issue.. Will fix it.